### PR TITLE
Handle interrupted wallet creation

### PR DESCRIPTION
**Pull request by brunobar79**
- Mark the wallet as damaged to prevent funds to be deposited
- Attempt to restore while tapping on version stamp

To repro the interrupted wallet creation scenario, you need to:
1 -  clear the keychain
2 - Add   `return ;` in the line 569 of `src/model/wallet.ts`
3 - Restart the app
4 - Tap on "Get a new wallet"

### CODE REVIEW

**Review by skylarbarrera — APPROVED** [review]
tested on sim & physical device in release. LGTM!! 🌈

File: src/hooks/useWalletsDebug.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  selectedWalletKey,
+} from '../utils/keychainConstants';
+import useInitializeWallet from './useInitializeWallet';
+import useWallets from './useWallets';
+import { colors } from '@rainbow-me/styles';
+import logger from 'logger';
+
+const keysOnly = keysWithValues => map(keysWithValues, item => item?.username);
+
+export default function useWalletsDebug() {
```
Comment (osdnk) [inline src/hooks/useWalletsDebug.js:33]: so nice

**Review by osdnk — APPROVED** [review]
looks more than awesome :)

File: src/hooks/useWalletsDebug.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+          if (!seed) {
+            throw Error('Missing seed');
+          }
+          const itemKey = seed.username.split(`_${seedPhraseKey}`);
+          const id = itemKey[0];
+          const seedObject = JSON.parse(seed.password);
+          const type = identifyWalletType(seedObject.seedphrase);
+
+          const currentWallet = {
+            addresses: [
```
Comment (osdnk) [inline src/hooks/useWalletsDebug.js:62]: can we extract this to some const?

**Review by jinchung — APPROVED** [review]

**Pull request merged**